RubberduckTests/Inspections/MemberNotOnInterfaceInspectionTests.cs

```diff
@@ -255,6 +255,70 @@ public void MemberNotOnInterface_Ignored_DoesNotReturnResult()
             }
         }
 
+#region https://github.com/rubberduck-vba/Rubberduck/issues/3343
```

Comment:
not sure I like the use of region for that...

Reply:
I think you're right. It helped when I was debugging them, but I should have taken it out.